### DISCUSSION TITLE
schedule / cron style

### DISCUSSION THREAD

**Discussion by mustafa0x** [RealOrangeOne/django-tasks#52] [Q&A] [answered]
This project seems focused on the traditional enqueuing and processing use case.

I thought I'd share my needs. Perhaps it can be considered for a future version. My primary use case is to execute a task at low intervals. The task processes db records added by services. To achieve that, I am currently using https://github.com/dbader/schedule.

Thanks.

**Comment by RealOrangeOne** [comment, marked as answer]
Cron syntax is definitely something I want to add, but not in the initial version due to the added complexity.

There's already a discussion open about cron-based scheduling https://github.com/RealOrangeOne/django-tasks/discussions/8, so I'm closing this in favour of that.
